File: development/NET/samples/server/Console/VB/MyRequest.vb
Imports System
Imports Softing.OPCToolbox.Server

Namespace Console
	Public Class MyRequest
		Inherits DaRequest
#Region "Constructor"
		'-----------------

		Public Sub New(ByVal transactionType As EnumTransactionType, ByVal sessionHandle As UInt32, ByVal aDaAddressSpaceElement As DaAddressSpaceElement, ByVal propertyID As Integer, ByVal requestHandle As UInt32)
			MyBase.New(transactionType, sessionHandle, aDaAddressSpaceElement, propertyID, requestHandle)
		End Sub

		'--
#End Region

	End Class
End Namespace

File: development/NET/samples/server/UpdateMode/CS/MyRequest.cs
using Softing.OPCToolbox.Server;

namespace UpdateMode
{
	public class MyRequest : DaRequest
	{
		#region Constructor
		//-----------------

		public MyRequest(
			EnumTransactionType transactionType,
			uint sessionHandle,
			DaAddressSpaceElement aDaAddressSpaceElement,
			int propertyID,
			uint requestHandle)
			:
			base(
				transactionType,
				sessionHandle,
				aDaAddressSpaceElement,
				propertyID,
				requestHandle)
		{

		}   //	end constructor

		//--
		#endregion

	}   //	end MyRequest class
}   //	ens namespace
